Src/VsVimShared/Implementation/IntelliCode/IntelliCodeExtensionAdapter.cs

```diff
@@ -0,0 +1,29 @@
+using Microsoft.VisualStudio.Text.Editor;
+using System.ComponentModel.Composition;
+
+namespace Vim.VisualStudio.Implementation.IntelliCode;
+
+[Export(typeof(IExtensionAdapter))]
+internal sealed class IntelliCodeExtensionAdapter : VimExtensionAdapter
+{
+    internal const string DifferenceViewerWithAdaptersRole = nameof(DifferenceViewerWithAdaptersRole);

```

Comment:
if this PR merges, please add a comment to our implementation that links to this PR & says vsvim is depending on this name.